packages/app/src/__deprecated__/renderer/locales/default/en-US.json

```diff
@@ -420,6 +419,28 @@
   "module.messages.sendButtonTooltipDescription": "Send message",
   "module.messages.textAreaPlaceholder": "Type your message",
   "module.messages.unreadOnly": "Unread only",
+
+  "module.messages.deletedThread": "1 conversation was deleted.",
+  "module.messages.deletedThreads": "{number} conversations were deleted.",
+  "module.messages.deleteThreadModalTitle": "Delete Conversations",
+  "module.messages.deleteThreadModalBody": "Do you really want to delete <b>{num, plural, =-1 {all conversations} one {# conversation} other {# conversations}}?</b>",
+  "module.messages.deleteThreadModalCancel": "Cancel",
+  "module.messages.deleteThreadModalAction": "Delete",
+  "module.messages.deletingThreadModalTitle": "Deleting Conversations",
+  "module.messages.deletingThreadModalSubtitle": "Deleting in progress...",
+  "module.messages.deleteThreadModalErrorTitle": "Deleting Conversations",
```

Comment:
(Not related to this line)

The file `packages/app/src/__deprecated__/main/default-news.json` should not be included in this PR.

Reply:
https://github.com/mudita/mudita-center/pull/827/commits/f5cce241774a21931a04a54278475bd7e3ffa4ae